### DISCUSSION TITLE
Error while new installation on debian?

### DISCUSSION THREAD

**Discussion by Feldi77** [python-telegram-bot/python-telegram-bot#2357] [Q&A]
I started a new installation on debian 9.

```
pip3 install python-telegram-bot --upgrade
pip3 install -r requirements.txt
```

Then I created the echobot.py from the examples via nano.

Error message:
```
root@v2116:~# python3 echobot.py
Traceback (most recent call last):
  File "echobot.py", line 21, in <module>
    from telegram import Update
ImportError: cannot import name 'Update'
root@v2116:~#
```

Is there a simple explanation or solution to fix this problem?

Best regards Christian

**Comment by Bibo-Joshi** [comment]
Please double check that you have no other package installed that provides a `telegram` module. Also double check if you installed PTB to a virtual environment that you need to activate before running echobot.

**Comment by Feldi77** [comment]
Hello,

I have done it exactly as documented. I installed everything in a virtual environment and ran the requirements.txt.
Already the example echobot.py throws the following error:

```
(app) root@v2116:~/telegram/app/bin# python3 echobot.py
Traceback (most recent call last):
  File "echobot.py", line 21, in <module>
    from telegram import Update
ImportError: cannot import name 'Update'
```

The installed libraries are as follows:

```
Package             Version
------------------- ---------
APScheduler         3.6.3
certifi             2020.12.5
cffi                1.14.4
cryptography        3.2.1
decorator           4.4.2
pip                 20.3.4
pycparser           2.20
python-telegram-bot 13.1
pytz                2021.1
setuptools          50.3.2
six                 1.15.0
telegram            0.0.1
tornado             6.1
tzlocal             2.1
wheel               0.36.2
(app) root@v2116:~/telegram/app/bin#
```

Is it possible that it depends on the installed python version. Currently I use a virtual machine with python 3.5

**Reply by Bibo-Joshi** [reply]
Ah, yes. As you can see in the Readme and on PyPi, py3.5 is not supported. It reached end of life in 09/2020 and we dropped it in order to include new py3.6+ only features.
